### DISCUSSION TITLE
辨識木瓜熟度的AI應用

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS3073-C/Python-Community-GS3073-C#91] [General]
新聞網址：https://news.sina.com.tw/article/20211029/40410482.html
題目網址：https://aigo.org.tw/zh-tw/competitions/details/404

用影像辨識加上聊天機器人，可以做一個辨識木瓜熟度的AI應用

AI做的事情大致上分成兩類：回歸、分類

辨識木瓜熟度的應用屬於分類的應用

**Comment by bonny123865** [comment]
判斷木瓜成熟居然需要千張以上的照片!
但長輩判斷木瓜成熟都是用壓的哈哈

**Comment by ccathyy** [comment]
現今AI做到先前的人們完全不敢想像的事
好期待之後科技能進步到哪種令現在的我意料不到的境界~

**Comment by Jay-722** [comment]
看能不能超越阿嬤們的智慧，我支持阿嬤

**Comment by chia-kai-chang** [comment]
使用線上的no-code服務，**不用寫程式**，就可以判斷不同成熟度的香蕉，也可以用手機玩

參考文章：https://medium.com/@warronbebster/teachable-machine-tutorial-bananameter-4bfffa765866
Google Teachable Maching：https://teachablemachine.withgoogle.com/

![image](https://user-images.githubusercontent.com/1768891/141449132-d56362f7-0d5c-4651-817d-48aa61a14666.png)


![image](https://user-images.githubusercontent.com/1768891/141449190-9d3cad00-acf6-4684-8a20-0937e5da67be.png)


![image](https://user-images.githubusercontent.com/1768891/141449316-4a3f513b-9753-46cb-b500-06d8b8a2b4b9.png)

**Comment by briankuo123** [comment]
希望以後可以用AI辨識水果甜不甜，不然常常有需多賣像很好的水果吃起來其實都沒什麼味道。

**Reply by pocper1** [reply]
可是這樣對水果商有壞處 買家都會依照判斷出來的結果（非常甜）才購買水果，造成大量水果滯銷（那些不甜的水果）因而報廢這些農作物

**Comment by ghost** [comment]
但是這個看外表就會讓我想到"催熟"的部分 以香蕉為例 如果香蕉外皮是催熟的 但裡面是生的 那也不見得是好事(果然老一輩的方法比較堪用

**Comment by Shiba010** [comment]
這項技術在現在看起來發展已經很成熟了！
